app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardDatastoresStep/components/DatastoresStepForm/DatastoresStepForm.js

```diff
@@ -265,13 +267,18 @@ class DatastoresStepForm extends React.Component {
 
     return (
       <div className={classes}>
+        <p style={{ marginLeft: -40 }}>
+          {__(
+            'Select source cluster(s) and a target cluster and click Add Mapping to add the mapping. Multiple mappings can be added.') // prettier-ignore
```

Comment:
@michaelkro if yah can update this as per @vconzola comment, we can merge this 🐶 !

Reply:
Lol, oops